app/models/components/system/admin/system_announcements_ability_component.rb

```diff
@@ -0,0 +1,17 @@
+module System::Admin::SystemAnnouncementsAbilityComponent
+  include AbilityHost::Component
+
+  def define_permissions
+    if user
```

Comment:
Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.